app/forms/DeclarationPage.scala

```diff
@@ -16,4 +16,18 @@
 
 package forms
 
-trait DeclarationPage
+import models.DeclarationType.{CLEARANCE, DeclarationType}
+import models.viewmodels.TariffContentKey
+
+trait DeclarationPage {
+  def defineTariffContentKeys(decType: DeclarationType): Seq[TariffContentKey] = Seq.empty
```

Comment:
What do you think about making this method abstract? To enforce implementation when creating a new page?

Reply:
Not all pages require tariff links so would force them to implement a method that they have no use for.